lessons/module-3/react-2-the-how.md

```diff
@@ -626,7 +609,7 @@ We now want to iterate through our array and create a Card component, passing it
 ```js
 // Ideas.js
 
-const Ideas = ({ideas}) => {
+function Ideas({ideas}){
```

Comment:
Because we're destructuring here (as we are in our `import` statements), it might be good to use consistent syntax and includes spaces around `ideas`, like so:

```function Ideas({ ideas }) {```